README.rst

```diff
@@ -477,6 +477,13 @@ This configuration file defines two *targets*: ``master`` and ``experimental``.
 synchronises all the public images with properties *nid* and/or *type* defined. The last one only
 synchronises images with ``type=baseimages``
 
+Installation with Docker
+------------------------
+By using the Dockerfile, it is possible to generate the image
+  docker build -t fiware-glancesync .
```

Comment:
Try to leave a white space after and before the command line and put the following content:

.. code::
    docker build -t fiware-glancesync .
 

and

..code::
    docker run -ti -e OS_USERNAME={OS_USERNAME} -e OS_PASSWORD={OS_PASSWORD} -e OS_AUTH_URL={OS_AUTH_URL} -e OS_TENANT_NAME={OS_TENANT_NAME} -e OS_REGION={OS_REGION} fiware-glancesync